app/views/subjects/index.html.erb

```diff
@@ -584,6 +584,38 @@
     <li><a data-value="twilight">Twilight</a></li>
     <li><a data-value="night">Night</a></li>
 </ul>
+<ul id="drop-dataset" class="tiny content f-dropdown" data-dropdown-content>
+    <li><a data-value="vr1_entire_dataset">Entire dataset</a></li>
+    <li><a data-value="vr1_daytime">Daytime</a></li>
+    <li><a data-value="vr1_nighttime">Nighttime</a></li>
+    <li><a data-value="vr1_blank">Blank</a></li>
+    <li><a data-value="vr1_non_blank">Non-blank</a></li>
+    <li><a data-value="vr1_one_animal">One animal</a></li>
+    <li><a data-value="vr1_simple">Simple (2-5 animals)</a></li>
+    <li><a data-value="vr1_complex">Complex (6+ animals)</a></li>
+    <li class="divider"></li>
+    <li><a data-value="vr1_single_species">Single species</a></li>
+    <li><a data-value="vr1_only_buffalo">Only buffalo</a></li>
+    <li><a data-value="vr1_only_elephant">Only elephants</a></li>
+    <li><a data-value="vr1_only_ostrich">Only ostriches</a></li>
+    <li><a data-value="vr1_only_warthog">Only warthogs</a></li>
+    <li><a data-value="vr1_only_wildebeest">Only wildebeest</a></li>
+    <li><a data-value="vr1_only_other">Only a single species, not listed above</a></li>
+    <li class="divider"></li>
+    <li><a data-value="vr1_multi_species">Multiple species</a></li>
+    <li><a data-value="vr1_multi_including_buffalo">Multiple species, including buffalo</a></li>
+    <li><a data-value="vr1_multi_including_elephant">Multiple species, including elephants</a></li>
+    <li><a data-value="vr1_multi_including_ostrich">Multiple species, including ostriches</a></li>
+    <li><a data-value="vr1_multi_including_warthog">Multiple species, including warthogs</a></li>
+    <li><a data-value="vr1_multi_including_wildebeest">Multiple species, including wildebeest</a></li>
+    <li><a data-value="vr1_multi_including_none_of_the_five">Multiple species, including not listed above</a></li>
+</ul>
+<ul id="drop-light" class="tiny content f-dropdown" data-dropdown-content>
```

Comment:
not sure if this is my error from earlier, or if you have added it, but looks like maybe we have two copies of the light dropdown here?